source/appModules/foobar2000.py

```diff
@@ -29,28 +45,47 @@ def event_gainFocus(self, obj, nextHandler):
 
 	def getElapsedAndTotal(self):
 		if not self.statusBar: return None

```

Comment:
There is still a case of returning a single `None` value. I would suggest returning a `namedTuple` or at least a pre-initialized return value: `ret = (None, None)` better would be `ret = collections.namedtuple('StatusBarTimes', 'elapsed total')(None, None)`

Then at every `return`, you return `ret`